internal/iomon/iomon.go

```diff
@@ -0,0 +1,150 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the LGPLv3, see LICENCE file for details.
+
+package iomon
+
+import (
+	"fmt"
+	"sync"
+	"time"
+	
+	"github.com/juju/utils/clock"
+
+	tomb "gopkg.in/tomb.v2"
```

Comment:
Regrouped. I've been going with goimports' default identifier recently, so I think I'll leave that.